packages/@angular/cli/utilities/app-utils.ts

```diff
@@ -0,0 +1,11 @@
+export function getAppFromConfig(apps: any, name: String) {
```

Comment:
This looks like a pretty good candidate for a static method on `models/config`. What do you think?